src/main/resources/application-dev.yml

```
@@ -101,4 +101,4 @@ unsplash:
 
 # CORS Configuration
 cors:
-  allowed-origins: ${CORS_ALLOWED_ORIGINS:http://localhost:3000}
\ No newline at end of file
+  allowed-origins: ${CORS_ALLOWED_ORIGINS:http://localhost:3000,https://twiggle.tech,https://*.vercel.app}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)